ingestr/src/google_ads/__init__.py

```
@@ -51,28 +52,28 @@ def google_ads(
         range_end="closed",
     )
     if report_spec is not None:
-        custom_report = Report().from_spec(report_spec)
+        custom_report, _ = Report.from_spec(report_spec)
         yield dlt.resource(
             daily_report,
             name="daily_report",
             write_disposition="merge",
-            primary_key=custom_report.primary_keys(),
+            primary_key=custom_report.primary_keys() + ["customer_id"],
```

Ruff (0.14.14)

Consider `[*custom_report.primary_keys(), "customer_id"]` instead of concatenation

Replace with `[*custom_report.primary_keys(), "customer_id"]`

(RUF005)